hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientExecutionServiceImpl.java

```diff
@@ -142,11 +168,11 @@ public void shutdown() {
         shutdownExecutor("internal", internalExecutor);
     }
 
-    private void executeInternalSafely(Runnable command) {
+    private void executeInternalSafely(Runnable command, ExecutionCallback executionCallback) {
         try {
             executeInternal(command);
```

Comment:
we are not trapping failures in executeInternal(command) to ExecutionCallback, I think this sould be using the submitInternal and then passing the callback with .andThen()